app/src/main/res/values/strings.xml

```diff
@@ -85,6 +85,7 @@
     <string name="successToast">Images successfully added.</string>
 
     <!-- Navigation drawer strings -->
+    <string name="img_to_pdf">Images to PDF</string>
```

Comment:
why is this change required?

Reply:
Because this is used as a key

Reply:
We dont need translations of keys, so we add keys in a constants file.
https://github.com/Swati4star/Images-to-PDF/blob/master/app/src/main/java/swati4star/createpdf/util/Constants.java

Reply:
Okay will change it then.